.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+### What happened?
+    Tell us what happened and provide as much details as possible, including logs.
+
+### What did you expect to happen?
+    Tell us about expected behaviour.
+
+### What is the GPU model?
+    Tell us about hardware configuration of the GPU.
```

Comment:
"Tell us about the hardware configuration of the GPU, including the output of 'nvidia-smi'"